app/views/moderations/index.html.erb

```diff
@@ -13,11 +13,9 @@
         <h2 class="crayons-subtitle-1">
           <%= @tag ? "##{@tag.accessible_name}" : t("views.moderations.all") %>
         </h2>
-        <% if FeatureFlag.enabled?(:moderator_role) %>
-          <div class="crayons-notice crayons-notice--warning">
-            The following moderator actions are temporarily unavailable in Mod Center: <strong>Unpublish Post</strong>, <strong>Unpublish All Posts</strong>, <strong>Flag User</strong> and <strong>Suspend User</strong>.
-          </div>
-        <% end %>
+        <div class="crayons-notice crayons-notice--warning">
+          The following moderator actions are temporarily unavailable in Mod Center: <strong>Unpublish Post</strong>, <strong>Unpublish All Posts</strong>, <strong>Flag User</strong> and <strong>Suspend User</strong>.
+        </div>
```

Comment:
This copy appears visible if the feature flag isn't enabled now that the FeatureFlag check was removed (i.e. it will appear visible in DEV). 

From reading [this comment](https://forem-team.slack.com/archives/C03CHBBMTJP/p1658767875871379) it seems like it should be behind the FeatureFlag, or is it my mistake in interpreting it that way?

Reply:
I think we need this text to be visible regardless of feature flag, because the related actions are going to be immediately "live" regardless of the feature flag, right? As in, without enabling the feature flag, we see "unpublish all posts, new flag user flow, etc" on the article page but not in the mod centre. The feature flag is only currently guarding the ability for an admin to assign anyone the new `moderator` role.

But I agree the slack message contradicts this, so I definitely think it's worth clarifying with Mai.

Reply:
I think I’m confused so please halp! 

With the feature flag enabled, we have the new role, the updated mod component (changed on article and mod center), and the copy, right?

With it disabled, there’s not new mod role, no copy, and no change to the component’s functionality on the article page or the mod center (meaning because the feature flag is disabled, it works as it currently does on production right now).

Reply:
@mirie @fdocr @aitchiss 

Okay! I see the mis-communication. 😅  Let me address.

Forget about the `super-moderator` role. Trash that concept (for the duration of this comment).

We presently have 2 user roles that can moderate articles: Trusted and Admin.

Trusted can flag a user. Admin can flag a user, unpublish a single post, unpublish all posts, and suspend a user.

These 4 actions are working in the Article view, but NOT working in Mod Center. To mitigate this, we are hiding these 4 actions in Mod Center only, so we NEED to inform Trusted and Admin users about their unavailability in Mod Center. Hence the warning banner.

These actions ARE NOT controlled by the `super-mod` FeatureFlag. We are hiding these actions regardless. The FeatureFlag ONLY allows the `super-mod` role to be assigned. It has NOTHING to do with the availability of the 4 actions.

I hope this helps! 🙏🏾

Reply:
Question: is this functionality that’s to be hidden on the mod center currently not working?

Reply:
This does help clarify this a lot from my end, thanks @msarit! I do have one follow up question:

Regardless if the FeatureFlag is ON/OFF, does it make sense to guard this copy with something like `if policy(Article).elevated_user?` (or something along those lines) so trusted users don't see this banner? 

It's just what came to mind, since `trusted` users don't have access to these features altogether, only admins and eventually moderators (whether FeatureFlag is ON/OFF would work the same).

Reply:
Oh well, sorry, one of the action is in fact available for `trusted` users... So my follow up can be disregarded though since looking back at it feels a bit overly specific!

I maintain the LGTM review 👍🏼

Reply:
Ok I re-read what you wrote @Arit. Okay, that whole component was re-factored...hmmm, so sounds like this work is partially flagged. Okay, I think my misunderstanding here is that I'm more used to the feature flag controlling/toggling the entire change put forth -- sounds like we use it a bit differently here.

Reply:
@mirie Currently, the functionality is working in Mod Center. The refactors that happened during the project caused them to stop working in Mod Center. Once I merge this PR, they will stop working in Mod Center.

Everything is working in Article page, both pre and post PR merge.

Reply:
I know less about the details of the implementation re: policies, etc so I trust in @fdocr and @aitchiss' advice on how to address.

However, I will say, let's timebox this to an hour at most. If we can't resolve, remove copy and I will work with @michael-tharrington on how to best communicate the issue to folks using the mod center actively